pkg/cloudstorage/manager.go

```diff
@@ -17,7 +17,7 @@ type CloudDriveConfig struct {
 }
 
 // Config is the cloud storage configuration for a node
-// TODO: This struct should eventually move under StorageNodeStatus
+// TODO: This struct should eventually move under StorageNode
```

Comment:
Just noticed this. Looks like it's a good time to move

Reply:
Yes if this gets moved, then you don't need to do the conversion from StorageNodeCloudDriveConfig and CloudConfig structs

Reply:
it causes import loop. Will check if I can fix it quickly